lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -100,6 +112,12 @@
     description:
       - Repo plugin type to use (i.e. C(rpm), C(docker)).
     default: rpm
+  repoview:
+    description:
+      - Enables repoview of published repository to improve browsing experience
```

Comment:
I think that `enable` indicates that this option can only enable the repoview and not delete it, but it looks like all of the code will still work to remove it as well (and it should). Looking at other modules, the phrasing is more often along the lines of `Whether to generate repoview files` or `Manage distributor repoview property`

Reply:
Sounds good. Pushed a fix.